README.md

```diff
@@ -120,6 +120,60 @@ _Note: Your application must be public to test the billing process. To test on a
     has_been_billed = activated_charge.status == 'active'
     ```
 
+### Session tokens
+
+The Shopify Python API library provides helper methods to decode [session tokens](https://shopify.dev/concepts/apps/building-embedded-apps-using-session-tokens). You can use the `get_decoded_session_token` function to help extract and decode a session token from an HTTP Authorization header.
+
+#### Basic usage
+
+```python
+from shopify import session_token
+
+decoded_payload = session_token.get_decoded_session_token(
+    authorization_header=your_auth_request_header,
+    api_key=your_api_key,
+    secret=your_api_secret,
+)
+```
+
+#### Use session_token to create a decorator
+
+Here's a sample use-case of building a Django decorator using `session_token`:
```

Comment:
If we're providing a Django example here, we should also be providing a Flask example. The library is supposed to be framework agnostic and the usage should cover popular frameworks.